MBBSEmu/MBBSEmu.csproj

```diff
@@ -11,6 +11,10 @@
     <None Remove="Assets\commandLineHelp.txt" />
     <None Remove="Assets\crashDetailsAPI.txt" />
     <None Remove="Assets\login.ans" />
+    <None Remove="Assets\MBBSEMU.DLL" />
```

Comment:
We should move these to the Test project so they're not included in the embedded resources for the packaged executables